admin/pg12/upgrade-to-pg12

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -e
+
+docker-compose down
+docker-compose \
+    -f docker-compose.yml \
+    -f admin/pg12/pg-9.5-stopped.yml \
+    up -d
```

Comment:
This will ignore any override files defined via `COMPOSE_NAME` in `.env` file. That looks right even for dev setup.